main.js

```diff
@@ -1 +1,188 @@
+  } else if (buttonStudy.classList[1] !== undefined) {
+    return 'Study';
+  } else if (buttonMeditate.classList[1] !== undefined) {
+    return 'Meditate';
+  }
+}
+
+function changeColorOrange() {
+    buttonExercise.classList.toggle('orange');
+    imageExercise.classList.toggle('hidden');
+    imageExerciseActive.classList.toggle('hidden');
+    unselectButtons(imageStudy, imageStudyActive, buttonStudy, 'green');
+    unselectButtons(imageMeditate, imageMeditateActive, buttonMeditate, 'purple');
+    catSelectionError(buttonExercise);
+}
+
+function changeColorGreen() {
+    buttonStudy.classList.toggle('green');
+    imageStudy.classList.toggle('hidden');
+    imageStudyActive.classList.toggle('hidden');
+    unselectButtons(imageExercise, imageExerciseActive, buttonExercise, 'orange');
+    unselectButtons(imageMeditate, imageMeditateActive, buttonMeditate, 'purple');
+    catSelectionError(buttonStudy);
+}
+
+function changeColorPurple() {
+    buttonMeditate.classList.toggle('purple');
+    imageMeditateActive.classList.toggle('hidden');
+    imageMeditate.classList.toggle('hidden');
+    unselectButtons(imageExercise, imageExerciseActive, buttonExercise, 'orange');
+    unselectButtons(imageStudy, imageStudyActive, buttonStudy, 'green');
+    catSelectionError(buttonMeditate);
+}
+
+function unselectButtons(selector1, selector2, border, color) {
+  if (selector1.classList.contains('hidden')) {
+    selector1.classList.remove('hidden');
+  }
+  if (!selector2.classList.contains('hidden')) {
+    selector2.classList.toggle('hidden');
+  }
+  if (border.classList.contains(color)) {
+    border.classList.toggle(color);
+  }
+}
+
+
+function buttonError() {
+  buttonStart.classList.toggle('backgroundColor');
+  setTimeout(function(){ buttonStart.classList.toggle('activityError'); }, 150);
+  setTimeout(function(){ buttonStart.classList.toggle('activityError'); buttonStart.classList.toggle('backgroundColor') }, 1700);
+  minuteError();
+  secondsError();
+  accomplishError();
+  activityFormError();
+}
+
+function minuteError() {
+  errorCheck(minutes, minError);
+}
+
+function secondsError() {
+  errorCheck(seconds, secError);
+}
+
+function accomplishError() {
+  accomplishErrorCheck(userAccomplish, accompError);
+}
+
+function catSelectionError(pressedButton) {
+  var button = pressedButton;
+  if (button === undefined || button.classList[1] === undefined) {
+    categoryError.classList.remove('invisibility');
+  } else if (!categoryError.classList.contains('invisibility')){
+    categoryError.classList.add('invisibility');
+  }
+
+}
+
+function activityFormError() {
+  if (buttonExercise.classList[1] === undefined && buttonStudy.classList[1] === undefined && buttonMeditate.classList[1] === undefined) {
+    categoryError.classList.remove('invisibility');
+  }
+}
+
+function errorCheck(inputField, errorMessage) {
+  var input = parseInt(inputField.value);
+  if (!input && input !== 0) {
+    errorMessage.classList.remove('invisibility');
```

Comment:
I believe this checks for non-numerical characters. maybe set inputField.value = "" and see if that fixes it?